pom.xml

```diff
@@ -79,6 +79,13 @@
       <version>2.0.2</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
```

Comment:
Gotta say I'm not a huge fan of annotation magic and this dependency in particular is somewhat of a pain to get setup with eclipse.